src/actions/api.js

```diff
@@ -1191,6 +1186,29 @@ export const onFetchProposalsVoteStatusByTokens = tokens => async dispatch => {
   }
 };
 
+export const onFetchBatchProposalsVoteStatus = (csrf, tokens) => dispatch => {
```

Comment:
Rename the action and don't pass the csrf as a parameter, use `withCsrf` internally as other functions currently do.
```suggestion
export const onFetchBatchProposalsVoteSummary = (tokens) => dispatch => {
```